src/RpcServer/RpcServer.Utilities.cs

```diff
@@ -13,14 +13,14 @@ partial class RpcServer
         private JObject ListPlugins(JArray _params)
         {
             return new JArray(Neo.Plugins.Plugin.Plugins
-                .OrderBy(u => u.Name)
+                .OrderBy(u => u.Name).Where(u => !(u is ILogPlugin))
```

Comment:
All `ILogPlugin` will not be displayed? I think it's incorrect.

Reply:
> All `ILogPlugin` will not be displayed? I think it's incorrect.

You are right, my intention is to exclude SystemLog. 
refs: https://github.com/neo-project/neo-node/blob/59da50115537c1941eee9365ae79febe7883146c/neo-cli/CLI/MainService.Plugins.cs#L109-L110

Reply:
Maybe you can exclude them if his assembly is `neo-node`